lib/src/base-styles/_nx-lists.scss

```diff
@@ -65,6 +66,22 @@
     padding: 6px 10px 6px 0;
     white-space: normal;
   }
+
+  .nx-list__item {
+    list-style-type: disc;
+
+    .nx-list {
+      margin-top: $nx-spacing-sm;
```

Comment:
This is suspicious.  Shouldn't this be handled by our normal margin and padding rules.  E.g, shouldn't `.nx-list` have a standard amount of top margin, which, when it's the first child in an `.nx-list__item`, gets removed, and then separately `.nx-list__item` has a certain amount of top padding?

Reply:
> This is suspicious. Shouldn't this be handled by our normal margin and padding rules. E.g, shouldn't `.nx-list` have a standard amount of top margin, which, when it's the first child in an `.nx-list__item`, gets removed, and then separately `.nx-list__item` has a certain amount of top padding?

If `.nx-list__item` has top padding it will appear above the text of the list item, not between that text and the embedded `.nx-list`. I think it's clearer it you look at the example HTML.

Reply:
> If .nx-list__item has top padding it will appear above the text of the list item, not between that text and the embedded .nx-list. I think it's clearer it you look at the example HTML.

Right, after looking at the example I'm back to my same question, but focusing on a standard margin being the correct solution.  Why doesn't `.nx-list` have a top margin by default?